addons/sale_product_configurator/tests/test_sale_product_configurator_ui.py

```diff
@@ -148,6 +148,8 @@ def test_05_product_configurator_pricelist(self):
         # Remove tax from Conference Chair and Chair floor protection
         self.product_product_conf_chair.taxes_id = None
         self.product_product_conf_chair_floor_protect.taxes_id = None
+        # YTI FIXME: This tour requires sale_management, as the Sale
+        # root menu item is clicked
```

Comment:
Yep, it's been known for quite some time, but can stay like this for the moment :D